enterprise/models.py

```diff
@@ -1244,6 +1250,24 @@ def _complete_user_enrollment():
                         discount_percentage=enrollment.discount_percentage,
                         sales_force_id=enrollment.sales_force_id,
                     )
+
+                    # If the pending enrollment was using a subscription license, generate a license enrollment
+                    # alongside the enterprise enrollment.
+                    if enrollment.license_uuid:
+                        source = EnterpriseEnrollmentSource.get_source(EnterpriseEnrollmentSource.ENROLLMENT_URL)
+                        enterprise_course_enrollment = EnterpriseCourseEnrollment.objects.get(
```

Comment:
I don't understand something here: does the enterprise_customer_user.enroll function called above not return the successful enrollment? I don't think you should have to query for it again, I think you should already have a reference to it. That would save you the call to source as well.

Reply:
Resolved: enroll uses the API instead of a DB call and does not return the enrollment)